clippy_lints/src/utils/conf.rs

```diff
@@ -356,10 +356,10 @@ define_Conf! {
     ///
     /// The maximum size of objects (in bytes) that will be linted. Larger objects are ok on the heap
     (too_large_for_stack: u64 = 200),
-    /// Lint: ENUM_VARIANT_NAMES.
+    /// Lint: ENUM_VARIANT_NAMES, STRUCT_FIELD_NAMES.
     ///
-    /// The minimum number of enum variants for the lints about variant names to trigger
-    (enum_variant_name_threshold: u64 = 3),
+    /// The minimum number of enum variants or struct fields for the lints about variant and field names to trigger
+    (item_name_threshold: u64 = 3),
```

Comment:
This will need a `#[conf_deprecated]` entry if it's being renamed, though it would also be reasonable to have a separate config variable for structs and enums to avoid that

Reply:
I've splitted the option to avoid having to deprecate the other one